<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPU_MEMORY_Dialog</class>
 <widget class="QDialog" name="CPU_MEMORY_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border-radius:10px;
	border-width:2px;
	
	background-color: rgb(107, 236, 255);
}

QPushButton:hover{
	
	background-color: rgb(105, 255, 88);
}</string>
  </property>
  <widget class="QPushButton" name="button_CPUMM_return">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>回到主页</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>81</y>
     <width>431</width>
     <height>335</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-width:2px;
	border-radius:10px;
	background-color: rgb(194, 255, 235);
	font: 14pt &quot;华文行楷&quot;;
}

QLabel:hover{
	
	background-color: rgb(255, 184, 234);
}


</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>381</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>CPU信息</string>
    </property>
    <widget class="QLabel" name="label_cpuUsage">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>311</width>
       <height>201</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>201</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>426</y>
     <width>391</width>
     <height>331</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border-width:2px;
	border-radius:10px;
	background-color: rgb(194, 255, 235);
	font: 14pt &quot;华文行楷&quot;;
}

QLabel:hover{
	
	background-color: rgb(255, 184, 234);
}</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>内存信息</string>
    </property>
    <widget class="QLabel" name="label_mmUsage">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>311</width>
       <height>191</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>191</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignJustify</set>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="button_accelerate">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>46</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加速</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>9</y>
     <width>611</width>
     <height>751</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 15, 71);</string>
   </property>
   <widget class="QLabel" name="label_rocket">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
